fuel/transformers/__init__.py

```diff
@@ -60,6 +64,24 @@ def get_epoch_iterator(self, **kwargs):
         self.child_epoch_iterator = self.data_stream.get_epoch_iterator()
         return super(Transformer, self).get_epoch_iterator(**kwargs)
 
+    def get_data(self, request=None):
+        if self.batch_input:
+            return self.get_data_from_batch(request)
+        else:
+            return self.get_data_from_example(request)
+
+    def get_data_from_example(self, request=None):
+        raise NotImplementedError(
+            str(type(self)) +
```

Comment:
We generally prefer to use formatting instead of string concatenation, it looks a bit cleaner IMHO. Something slightly more informative would be good too e.g. ```"`{}` does not support examples as inputs, but `batch_input` was set to `False`".format(type(self))```